python/setup.py

```diff
@@ -47,7 +47,7 @@
                 'cxx/well.cpp',
                 'cxx/export.cpp'
         ],
-        libraries=['opmcommon', 'boost_filesystem', 'boost_regex', 'ecl', 'z'],
```

Comment:
The 'z' was a transitive dependency from libecl.

Reply:
Very good.  I'll try to remove that too and rerun the build check.

Reply:
Build goes through without either `ecl` or `z`.